frontend/integration-tests/protractor.conf.ts

```diff
@@ -92,6 +92,7 @@ const testSuites = {
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts',
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts',
   ]),
+  'ceph-storage-plugin': suite(['tests/storage-dashboard.scenario.ts']),
```

Comment:
This should be defined in `ceph-storage-plugin` plugin folder. Please take a look at `packages/kubevirt-plugin/package.json` for inspiration how to define a test suite there.

There is `integrationTestSuites` under `consolePlugin` field, in your case, it should look like this:
```
"integrationTestSuites": {
      "ceph-storage": ["integration-tests/**/*.scenario.ts"]
}
```

Reply:
Fixed